update/Pos/Update/Poll/Types.hs

```diff
@@ -211,6 +213,9 @@ data PollModifier = PollModifier
     , pmActiveProps       :: !(MapModifier UpId ProposalState)
     , pmDelActivePropsIdx :: !(HashMap ApplicationName (HashSet UpId))
     , pmSlottingData      :: !(Maybe SlottingData)
+    , pmEpochProposers    :: !(Maybe (HashSet StakeholderId))
+    -- ^ Cache of DB state.
```

Comment:
@gromakovsky it is not optimization at all, it's kostil to not get it from DB during run of transformer, because it is counterintuitive change PollModifier during running